deploy/docs/Upgrading_with_Helm.md

```diff
@@ -0,0 +1,85 @@
+
+# Upgrading with Helm
+
+When a new version of a chart is released, or when you want to change the configuration of your release, you can use the helm upgrade command.
+
+This command upgrades a release to a specified version of a chart and/or updates chart values.
+
+Required arguments are release and chart.
+
+The chart argument can be one of: - a chart reference; use `–version` flag for versions other than latest, - a path to a chart directory, - a packaged chart, - a fully qualified URL.
```

Comment:
Should the arguments be in a list? The markdown isn't rendering a list when I view these.

Reply:
fixed it.